messages/bitcoin

```diff
@@ -4,5 +4,6 @@ from pycoingecko import CoinGeckoAPI
 # list of coins https://api.coingecko.com/api/v3/coins/list
 price = CoinGeckoAPI().get_price(ids='bitcoin', vs_currencies='usd')
 
+print 'step:%d' % (0.09)
```

Comment:
I don't think this is going to do anything, since the code in unicorn-scroller.py doesn't interpret "step" as a command. Are you trying to set the scroll speed here, if so, I'm not sure it makes sense to do this for different messages?

Reply:
yes i forgot to push one more commit ;-) 
now solved

Reply:
I've not tried it but from reading the code, I don't think this works, since all of the commands are processed together for each message (in process_message), so the step will get set to 0.09 then immediately back to 0.05 - it won't have any effect.

How about if, instead of setting step directly, the message could set a multiplier to be applied to the step for the duration of that message?